release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -621,6 +621,91 @@ For more information, see xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-
+** Added the `PrometheusScrapeSampleLimitHit` alert to detect targets hitting the sample limit.
+
+* *Changed*
+** Fixed the `KubeDaemonSetRolloutStuck` alert to use the updated metric `kube_daemonset_status_updated_number_scheduled` from `kube-state-metrics`.
+** Replaced the `KubeJobCompletion` alert with `KubeJobNotCompleted`. The new `KubeJobNotCompleted` alert avoids false positives when an earlier job failed but the most recent job succeeded.
+** Updated the `NodeNetworkInterfaceFlapping` alert to exclude `tunbr` interfaces from the alert expression.
+
+[id=ocp-4-11-monitoring-enable-alert-routing-for-user-workload-momitoring]
+==== Enable alert routing for user workload monitoring
+A cluster administrator can now enable alert routing for user workload monitoring so that developers and other users can configure custom alerts and alert routing for their user-defined projects.
+
+[id=ocp-4-11-monitoring-enable-dedicated-alertmanager-for-user-defined-alerts]
+==== Enable a dedicated Alertmanager instance for user-defined alerts
+You can optionally enable a separate instance of Alertmanager dedicated to sending alerts only for user-defined projects.
+This feature can help reduce the load on the default platform Alertmanager instance and can better separate user-defined alerts from default platform alerts.
+
+[id=ocp-4-11-monitoring-use-additional-authenticaion-settings-for-remote-write]
+==== Use additional authentication settings for remote write configuration
+You can now use the following authentication methods to access a remote write endpoint: AWS Signature Version 4, custom Authorization header, and OAuth 2.0. 
+Before this release, you could only use TLS client and basic authentication.
+
+[id=ocp-4-11-monitoring-better-manage-promql-queries-in-web-console]
+==== Create, browse, and manage PromQL queries more easily in the web console
+The Query Browser on the *Observe* -> *Metrics* page of the {product-title} web console adds various enhancements to improve your ability to create, browse, and manage PromQL queries.
+For example, administrators can now duplicate existing queries and use autocomplete suggestions when building and editing queries.
+
+[id=ocp-4-11-monitoring-doubled-scrape-interval-for-servicemonitors-single-node]
+==== Doubled the scrape interval for ServiceMonitors in single node deployments
+The scrape interval has been doubled for all Cluster Monitoring Operator (CMO) controlled ServiceMonitors on single-node {product-title} deployments. 
+The maximum interval is now two minutes.
+
+[id=ocp-4-11-monitoring-create-alerting-rules-based-on-platform-metrics]
+==== Create alerting rules based on platform monitoring metrics (Technology Preview)
+This release introduces a Technology Preview feature in which administrators can create alerting rules based on existing platform monitoring metrics.
+This feature helps administrators to more quickly and easily create new alerting rules specific to their environments.
```

Comment:
See my earlier comments. It seems like whenever you discuss administrators, you don't directly address the user with second person. If your users aren't administrators, then you are correct in doing so. 

The occasional departure from second person does kind of throw me off though. To keep the user included in all sections, could you rephrase the last sentence to something like: 

`This feature helps administrators to more quickly and easily create new alerting rules specific to your development environments.` 

In short, right now, I'm concerned users will read through these sections and when they stop reading "you" and see "administrators` they'll think this content is not for them, which raises the question if the content should even be included.